src/yamlProjectConfiguration/v2.ts

```diff
@@ -138,6 +139,12 @@ function parseGenezioConfig(config: unknown) {
         frontend: zod.array(frontendSchema).or(frontendSchema).optional(),
     });
 
+    if (ENVIRONMENT === "dev") {
+        const regionValues = regions.map((region) => region.value);
+        const regionsEnum = [...regionValues, developmentRegion];
+        v2Schema.shape.region = zod.enum(regionsEnum as [string, ...string[]]).optional();
+    }
```

Comment:
Is allowing `dev-fkt` in YAML relevant if the region will be anyway discarded on the backend?

Reply:
Yes, it's not going to be discarded. I am using it to redirect deployments to the development server.
More context here: https://genezio.slack.com/archives/C042H2S0V7U/p1716302923162819